http/src/main/java/io/micronaut/http/hateos/JsonError.java

```diff
@@ -24,92 +24,28 @@
 import java.util.Optional;
 
 /**
- * A class that can be used to represent JSON errors that complies to Vnd.Error without the content type requirements.
+ * Deprecated. Please use io.micronaut.http.hateoas.JsonError
```

Comment:
Added "Deprecated..." as Javadoc for the existing classes, on top of the `@deprecated` annotation

Reply:
There is an `@deprecated` javadoc tag should use